DynamoDbEncryption/runtimes/java/src/main/java/software/amazon/cryptography/dbencryptionsdk/dynamodb/enhancedclient/DynamoDbEnhancedClientEncryption.java

```diff
@@ -134,12 +146,95 @@ private static DynamoDbTableEncryptionConfig getTableConfig(
             // Detect Encryption Flags that are Ignored b/c they are in a Nested Class
             scanForIgnoredEncryptionTags(topTableSchema, attributeName, path);
         }
+        return actions;
+    }
+
+    // given action maps from multiple tables, merge them into one
+    private static Map<String, CryptoAction> MergeActions(List<Map<String, CryptoAction>> actionList)
```

Comment:
```suggestion
    private static Map<String, CryptoAction> mergeActions(List<Map<String, CryptoAction>> actionList)
```

I'll stop leaving these, but let's do camelcase method names

Reply:
Where I come from, static and global things are camel case starting with a capital, and instance things are camel case starting with lowercase.